var/spack/repos/builtin/packages/magma/package.py

```diff
@@ -188,19 +188,38 @@ def cache_test_sources(self):
         install test subdirectory for use during `spack test run`."""
         self.cache_extra_test_sources([self.test_src_dir])
 
-    def test(self):
+    def test_magma(self):
```

Comment:
The new longer name of the test method doesn't convey what's done inside. `test_all` is better since all tests are performed inside (you're not splitting into multiple methods with `test_` prefix.

Reply:
Or perhaps `test_all_c`?